### PR TITLE
Update Readme intro #179

### PR DESCRIPTION
**Pull request by glebm**
* Removes links to abandoded forum engines.
* Lists current and planned features.
* Lists infrastructure dependencies.
* Removes the "Get your parent app ready" section, as everything is configured in the initializer now.

### CODE REVIEW

**Comment by glebm** [comment]
Rebased

File: README.mkdn [hunk 1]
```diff
@@ -14,12 +23,14 @@ is an engine - not a standalone app.
 
 [Discourse]: http://www.discourse.org/
 [Forem]: https://www.github.com/radar/forem
-[Tectura]: https://github.com/caelum/tectura
-[Heterotic Beast]: https://github.com/distler/heterotic_beast
-[Altered Beast]: https://www.github.com/courtenay/altered_beast
 
+Thredded works with SQLite, MySQL (v5.6.4+), and PostgreSQL.
+Thredded has no infrastructure dependencies other than the database and (optionally) an ActiveJob backend.
```
Comment (jayroh) [inline README.mkdn]: How come "optionally"?

Comment (glebm) [inline README.mkdn]: If there is no ActiveJob backend (such as Redis) set, ActiveJob will just run the jobs inline.

**Comment by glebm** [comment]
English isn't my native language, so I'd imagine this needs some grammar fixing.

File: README.mkdn (hunk 1 above)
Comment (jayroh) [inline README.mkdn]: How about?

> Thredded has no infrastructure dependencies other than the database and, if configured in the parent application, the ActiveJob backend dependency (redis, etc)

**Comment by jayroh** [comment]
Dude your English is JUST fine.

Better than most people in this country :)

File: README.mkdn (hunk 1 above)
Comment (glebm) [inline README.mkdn]: Done

**Pull request merged**